src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -1,5 +1,6 @@
 #include "bin/src/halide_hexagon_remote.h"
 #include <HalideRuntime.h>
+#include <HalideRuntimeHexagonHost.h>
```

Comment:
I just tried removing it, but found that it is still needed for the power mode enums.